content/_data/changelogs/weekly.yml

```diff
@@ -8784,7 +8784,6 @@
   - type: rfe
     category: rfe
     pull: 4911
-    issue: 63429
```

Comment:
Why?

Reply:
My mistake.  I didn't read the commit message heading of https://github.com/jenkinsci/jenkins/pull/4911/commits/992dc7d8785d26fc18f9c8ff357e3724c9d53b65 that specifically references JENKINS-63429.  @megathaum flagged it during her review that JENKINS-63429 seemed unrelated PR-4911.

I'll revert that change.  Thanks for catching it.

Reply:
Reverted in 3a768434ef5452a1ac34d4a08fe1e410fd815a07